src/map/pc.c

```diff
@@ -5320,6 +5320,9 @@ static int pc_useitem(struct map_session_data *sd, int n)
 	// Update item use time.
 	sd->canuseitem_tick = tick + battle_config.item_use_interval;
 
+	if (nameid == ITEMID_MEGAPHONE)
+		sd->state.using_megaphone = 1;
```

Comment:
If for any reason the `loudhailer()` command is not called, this flag is never unset (it could happen if the user  or teleports, uses the refresh command, steps into a warp portal or is recalled away through guild ecall, wedding skills, GM recall, etc)

Reply:
Fixed in https://github.com/HerculesWS/Hercules/pull/2775/commits/1d00e2268b7a20077d9d61dc8989b9f770344e59.